### PR TITLE
Add utility methods in AFS to manipulate project folders

### PR DESCRIPTION
**Pull request by amirakahya**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [X] The commit message follows our guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
This PR introduce new utility functions in AFS.

**What is the current behavior?** *(You can also link to an open issue here)*
Currently it is not possible to retrieve project folders directly from AppFileSystem instances or Project instances whereas it is the case for project files.

**What is the new behavior (if this is a feature change)?**
We added the same utility methods as the one available for project files for project folders.

### CODE REVIEW

File: afs/afs-core/src/test/java/com/powsybl/afs/AfsBaseTest.java [hunk 1]
```diff
@@ -204,4 +204,18 @@ public void moveToTest() {
...
 
+    @Test
+    public void findProjectFolderTest() {
+
+        Project project = afs.getRootFolder().createProject("test");
+        ProjectFolder test1 = project.getRootFolder().createFolder("test1");
+        ProjectFolder projectFolderResultat = afs.findProjectFolder(test1.getId());
+
+        assertNotNull(projectFolderResultat);
+        assertEquals(projectFolderResultat.getId(), test1.getId());
```
Comment (murgeyseb) [inline afs/afs-core/src/test/java/com/powsybl/afs/AfsBaseTest.java]: Two parts of assertions must be inverted ; test framework defines assertAquals arguments as:
assertEquals(Object expected, Object actual)

File: afs/afs-core/src/test/java/com/powsybl/afs/ProjectTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package com.powsybl.afs;
```
Comment (mathbagu) [inline afs/afs-core/src/test/java/com/powsybl/afs/ProjectTest.java]: You forget to add the license information. Change your IDE configuration not to forget it

File: afs/afs-core/src/main/java/com/powsybl/afs/Project.java [hunk 3]
```diff
@@ -56,4 +56,10 @@ ProjectFile createProjectFile(NodeInfo nodeInfo) {
             return new UnknownProjectFile(context);
         }
     }
+
+    public ProjectFolder createProjectFolder(NodeInfo nodeInfo) {
```
Comment (mathbagu) [inline afs/afs-core/src/main/java/com/powsybl/afs/Project.java]: Is this method really `public`? Do we really want to expose it through the API?

**Comment by geofjamg** [comment]
I close this PR as there is an issue with Sonar and external PR (PR from cloned repo). Another PR has to be recreated from a powsybl-core branch.

**Pull request closed without merging**